### ISSUE TITLE
Synchronization errors with gmail accounts in release 1.7.0

### ISSUE THREAD

**Issue by jesusdelariva** [Foundry376/Mailspring#1660]
I updated to the release 1.7.0 and I am showing synchronization errors with my gmail accounts, I have three and the three is failing synchronization, I already authenticated myself again, but still appearing synchronization errors. It was working fine with the previous version.

**Comment by chung5829** [comment]
same issues here. All my 4 gmails having issues since the update

**Comment by joy4eg** [comment]
Same for me.
```
525513 [2019-10-14 20:56:59.107] [background] [info] syncFolderChangesViaCondstore - Mailspring/Snoozed: modseq 1 to 1, uidnext 1 to 1
525513 [2019-10-14 20:56:59.108] [background] [info] Sync loop deleting unlinked messages with phase 2.
525513 [2019-10-14 20:56:59.109] [background] [info] Sync loop complete.
525513 [2019-10-14 20:57:11.642] [metadataExpiration] [info] Scanning for expired metadata
525513 [2019-10-14 20:57:11.642] [metadataExpiration] [info] -- Will wake for next expiration in 7200sec
525513 [2019-10-14 20:57:13.397] [calContacts] [critical] 
 ***
 *** A JSON exception occurred during program execution: 
 *** [json.exception.type_error.316] invalid UTF-8 byte at index 0: 0xA0
 ***

525513 [2019-10-14 20:57:13.406] [calContacts] [critical]  *** Stack trace (line numbers are approximate):
 *** (unknown)  clone
 ***

525560 [2019-10-14 20:57:13.618] [main] [info] Identity created at 1554727502 - using ID Schema 1

```

**Comment by RemyyB** [comment]
I also have this on SMTP emails

**Comment by bengotow** [comment]
Hey folks! thanks for reporting this — I'm investigating now. If you could take a look at Mailspring's logs (Developer > Show Mailsync Logs in the menu, and then look at the `mailsync-***.log` files), and see if they show the same error as @joy4eg's I'd appreciate it! Will get this fixed ASAP.

**Comment by zachschrock** [comment]
My GSUITE accounts do not work but my stand alone GMAIL does.

**Comment by joy4eg** [comment]
@zachschrock  I'm not sure but it can be related to contact syncing. My account is at mailbox.org. Check your logs.

**Comment by bengotow** [comment]
Hey folks! I was just able to reproduce this - it turns out I've been doing testing on primarily Google (@gmail.com) test accounts and this impacts Google Apps for Business. Will get this fixed. Thanks for the logs!

**Comment by bengotow** [comment]
Hey folks—just a heads up that the fix for this has shipped in 1.7.1. Decided to push it out along with the fix for #1659 because there have been quite a few reports of this happening via Twitter and our support channel. 

The app should auto-update in an hour or two, or you can choose "Check for Updates" from the menu or visit https://github.com/Foundry376/Mailspring/releases/tag/1.7.1 to get it now. Thanks - sorry for the hassle on this one!

**Comment by zachschrock** [comment]
Works perfect for me now, thanks!

**Comment by jesusdelariva** [comment]
Errors fixed with release 1.7.1, thanks!
